docs/REST API/rest-api.md

```diff
@@ -6936,12 +6938,74 @@ Deletes the session specified by ID. When the user associated with the session n
 
 ## Software
 
+- [Upload a software](#upload-a-software)
+- [Delete a software](#delete-a-software)
 - [List software titles](#list-software-titles)
 - [List software versions](#list-software-versions)
 - [Count software](#count-software)
 - [Get software title](#get-software-title)
 - [Get software version](#get-software-version)
 
+### Upload a software
+
+Upload a software to install on macOS and Windows hosts during enrollment.
+
+`POST /api/v1/fleet/software`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| software        | file    | form | **Required**. Installer file. PKG or DMG for macOS and MSI or EXE for Windows hosts.   |
```

Comment:
```suggestion
| software        | file    | form | **Required**. Installer file. PKG or DMG for macOS and MSI for Windows hosts.   |
```

Removing EXE for now.

For capitalization/format, I would check what we do elsewhere in the docs to be consistent. Do we use "PKG" or ".pkg" and ".msi" or "MSI"?

Reply:
As far as I remember we agreed to capitalize well known extensions (e.g. XML). So far it is used in couple of places in docs.